src/pages/Captures.tsx

```diff
@@ -1,12 +1,35 @@
-import CaptureRoot from 'components/capture/Root';
+import AddIcon from '@mui/icons-material/Add';
+import { Button, Toolbar } from '@mui/material';
+import CatalogList from 'components/CatalogList';
 import PageContainer from 'components/shared/PageContainer';
-import { Outlet } from 'react-router-dom';
+import { useState } from 'react';
+import { NavLink } from 'react-router-dom';
 
 const Capture = () => {
+    const [captureList] = useState([]);
+    const [isLoading] = useState(false);
+
     return (
         <PageContainer>
-            <CaptureRoot />
-            <Outlet />
```

Comment:
Since new capture is no longer a child we don't need the `Outlet` and we can just make the `CaptureRoot` directly handled here.